packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -138,6 +138,8 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		'data-type': name,
 		'data-title': blockTitle,
 		className: classnames(
+			// The wp-block className is important for editor styles.
+			'wp-block block-editor-block-list__block',
```

Comment:
wp-block shouldn't be added to blocks with the align wrapper, this broken full/wide alignments.